CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   by adding a `transformers` import.
 - Added safety checks for extracting tar files
 - Turned superfluous warning to info when extending the vocab in the embedding matrix, if no pretrained file was provided
+- `TrackEpochCallback` is now a `EpochCallback`.
```

Comment:
Can you move this up to the `## Unreleased` section?

Reply:
Sure!